framework/src/security/security_utils.c

```diff
@@ -267,11 +267,14 @@ int secutils_convert_path_s2h(const char *path, uint32_t *slot)
 	if (!strncmp(path, SS_PATH, sizeof(SS_PATH) - 1)) {
 		*slot = atoi(&path[3]);
 		return 0;
-	} else if (!strncmp(path, FACTORY_PATH, sizeof(FACTORY_PATH) - 1)) {
+	}
+#if (defined(CONFIG_ARCH_BOARD_ARTIK053S) || defined(CONFIG_ARCH_BOARD_ARTIK053)) \
+		&& defined(CONFIG_S5J_HAVE_SSS)
```

Comment:
`CONFIG_HAVE_XX` means it is supported by HW.
User selects it using `CONFIG_XX` so that for this case, `CONFIG_S5J_SSS` is right.

Reply:
Hmm.. FACTORYKEY_ARTIK_CERT actually needs H/W support. I brought this whole ifdef comes from other sources (ex. security_internal.h, line 25).

Reply:
Anyway this "else if" is deleted.